packages/generator/src/edmx-to-vdm/v2/function-import.ts

```diff
@@ -6,6 +6,9 @@ import { parseFunctionImports } from '../../edmx-parser/v2';
 import { ServiceMetadata } from '../../edmx-parser/edmx-file-reader';
 import { parseFunctionImportReturnTypes } from '../common/action-function-return-types';
 
+const extractResponse = (functionName: string) => (response: string) =>
+  `${response}.${functionName}`;
```

Comment:
Where does the name extractResponse come from? It look like this methods adjustes the value alias `val` with some prefix? Could we find a more descriptive name?